client/my-sites/plugins/plugin-details-CTA/usps.tsx

```diff
@@ -69,7 +69,9 @@ const USPS: React.FC< Props > = ( {
 					{
 						id: 'refund',
 						image: <Gridicon icon="refund" size={ 16 } />,
-						text: translate( '30 day money-back guarantee' ),
+						text: translate( '%(days)d-day money-back guarantee', {
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Money Back %(days)d-Day Guarantee' )` *ES Score: 9*